### PR TITLE
Draft Do not Merge - DT 1504 map management event

### PR DESCRIPTION
**Pull request by dnyanesh-sonwane**

### CODE REVIEW

**Review by mikehalmamoj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The specific events currently being raised are
 *   **OFFENDER_REGISTRATION_CHANGED** is raised when an offender's registration is changed. sourceId of the event is source id of the registration.
 *   **OFFENDER_REGISTRATION_DELETED** is raised when an offender's registration is deleted. sourceId of the event is source id of the registration.
 *   **OFFENDER_REGISTRATION_DEREGISTERED** is raised when an offender's registration is deregistered. sourceId of the event is source id of the deregistation.
+*   **OFFENDER_MANAGEMENT_TIER_CALCULATION_REQUIRED** is raised when an offender's management tier is changed. sourceId of the event is source id of the MANAGEMENT_TIER_EVENT.
```
Comment (andymarke) [inline README.md:30]: should be:  `is raised when an offender's management tier needs recalculating`

File: src/main/resources/swagger-description.html [hunk 2]
```diff
@@ -23,6 +23,7 @@ <h3>Generate events for the offender changes in probation</h3>
         <li><b>OFFENDER_REGISTRATION_CHANGED</b> is raised when an offender's registration is changed. sourceId of the event is source id of the registration</li>
         <li><b>OFFENDER_REGISTRATION_DELETED</b> is raised when an offender's registration is deleted. sourceId of the event is source id of the registration</li>
         <li><b>OFFENDER_REGISTRATION_DEREGISTERED</b> is raised when an offender's registration is deregistered. sourceId of the event is source id of the deregistation</li>
+        <li><b>OFFENDER_MANAGEMENT_TIER_CALCULATION_REQUIRED</b> is raised when an offender's management tier is changed. sourceId of the event is source id of the MANAGEMENT_TIER_EVENT</li>
```
Comment (andymarke) [inline src/main/resources/swagger-description.html:26]: same here

**Review by andymarke — CHANGES_REQUESTED** [review]
Just a correction on the documentation

File: README.md (hunk 1 above)
Comment (dnyanesh-sonwane) [reply README.md]: Done

File: src/main/resources/swagger-description.html (hunk 2 above)
Comment (dnyanesh-sonwane) [reply src/main/resources/swagger-description.html]: Done

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**